_posts/2022-05-12-Socratic-Seminar-30.md

```diff
@@ -37,3 +37,25 @@ BIP 119 is not new, but since talk of activation ramped up a lot this month we a
 <https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-23.0.md>
 
 One interesting new feature is [CJDNS](https://github.com/bitcoin/bitcoin/blob/23.x/doc/cjdns.md) support.
+
+## History of bitcoin-s consensus forks
+
+<https://blog.bitmex.com/bitcoins-consensus-forks/>
+
+A great overview of all the consensus forks, and the mechanism to activate that fork.
+
+## 7 next steps for BIP 119
```

Comment:
Should we just add these to the BIP 119 omnibus topics above?

Reply:
If you want to re-order feel free but otherwise seems like nit picking to me